docs/BuildExtensibility.md

```diff
@@ -0,0 +1,141 @@
+![UI5 icon](https://raw.githubusercontent.com/SAP/ui5-tooling/master/docs/images/UI5_logo_wide.png)
+
+# UI5 Build Extensibility
+
+The UI5 Tooling enables it to define a custom build for your UI5 project by adding additional tasks to the  [standard task list](https://github.com/SAP/ui5-builder/blob/master/README.md#tasks).
```

Comment:
> The UI5 Tooling enables it to

This sounds odd.

> by adding additional tasks to the standard task list

Custom tasks are not added to the standard task list. I.e. they do not become "standard tasks".

What about `The UI5 Build Extensibility enables a developer to enhance the build process of any UI5 project. In addition to the [standard task list](https://github.com/SAP/ui5-builder/blob/master/README.md#tasks), custom tasks can be created`

Reply:
Done.